crates/walrus-service/src/client/cli/runner.rs

```diff
@@ -1632,6 +1598,53 @@ impl ClientCommandRunner {
     }
 }
 
+pub struct StoreOptions {
```

Comment:
Looks like this doesn't need to be `pub`?

Reply:
Done in 0073b0725c36822f58471d8ddc9371ce64711781.